python/cudf/dataframe/dataframe.py

```diff
@@ -1542,6 +1543,57 @@ def partition_by_hash(self, columns, nparts):
         # Slice into partition
         return [outdf[s:e] for s, e in zip(offsets, offsets[1:] + [None])]
 
+    def replace(self, to_replace, value):
+        """
+        Replace values given in *to_replace* with *value*.
+
+        Parameters
+        ----------
+        to_replace : numeric, str, list-like or dict
+            Value(s) to replace.
+
+            * numeric or str:
+
+                - values equal to *to_replace* will be replaced
+                  with *value*
+
+            * list of numeric or str:
+
+                - If *value* is also list-like,
+                  *to_replace* and *value* must be of same length.
+
+            * dict:
+
+                - Dicts can be used to replace
+                  different values in different columns.
+                  For example, `{'a': 1, 'z': 2}` specifies
+                  that the value 1 in column `a` and the value
+                  2 in column `z` must be replaced with
+                  *value*.
+
+        value : numeric, str, list-like, or dict
+            Value(s) to replace `to_replace` with.
+            If a dict is provided,
+            then dict keys must match the keys in *to_replace*,
```

Comment:
Weird line break choice location, should only break when needed to to stay under character limit.

Reply:
Sorry! It's a force of habit picked up from here:

https://rhodesmill.org/brandon/2012/one-sentence-per-line/

Will change

Reply:
Fixed